util/windows/getopt/getopt.cpp

```diff
@@ -0,0 +1,698 @@
+/*
```

Comment:
Does this actually need to be a `.cpp` file?  From my quick skim of it, it looks like it's C, not C++, code.